network/transport/v2/protocol.go

```diff
@@ -71,22 +116,135 @@ func (p protocol) UnwrapMessage(envelope interface{}) interface{} {
 func (p protocol) Configure(_ transport.PeerID) {
 }
 
-func (p protocol) Start() {
+func (p *protocol) setupNatsHandler() error {
+	conn, err := p.natsConnectHandler(
+		p.config.Nats.Hostname,
+		p.config.Nats.Port,
+		time.Duration(p.config.Nats.Timeout)*time.Second,
+	)
+	if err != nil {
+		return err
+	}
+
+	js, err := conn.JetStream()
+	if err != nil {
+		return err
+	}
+
+	if _, err = js.Subscribe(events.PrivateTransactionsSubject, func(msg *nats.Msg) {
+		if err := p.handlePrivateTx(msg); err != nil {
+			logrus.Errorf("failed to handle private transaction: %v", err)
+
+			if err := msg.Nak(); err != nil {
```

Comment:
Can I brick a peer's node by sending a TX that causes an error in `handlePrivateTx`, so that it can process no subsequent private TXs?

Reply:
The NATS docs state that:

> Nak Signals that the message will not be processed now and processing can move onto the next message, NAK'd message will be retried

So I would hope that the next message won't be the NAK'd message but rather the actual next message in the stream.

Reply:
This requires a todo + issue for persistent retries

Reply:
See: https://github.com/nuts-foundation/nuts-node/issues/693